src/test/java/teammates/test/cases/action/InstructorFeedbackQuestionEditActionTest.java

```diff
@@ -1320,6 +1320,7 @@ public void testExecuteAndPostProcessRubricQuestion() {
                 Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_SUBQUESTION + "-1", "This student has tried his/her best.",
                 Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_CHOICE + "-0", "Yes",
                 Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_CHOICE + "-1", "No",
+                Const.ParamsNames.FEEDBACK_QUESTION_RUBRIC_WEIGHTS_ASSIGNED, "on",
```

Comment:
Take a look at other rubric test case. If `FEEDBACK_QUESTION_RUBRIC_WEIGHTS_ASSIGNED` is no there. There should be no `FEEDBACK_QUESTION_RUBRIC_WEIGHT` at all.